src/pages/viewCart/Cart.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from 'react';
+import '../viewCart/cart.scss';
+import { useGetCart } from './redux/hooks';
+
+function CartTable() {
+  const { handleGetCart, result } = useGetCart();
+  useEffect(() => {
+    handleGetCart();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+  return result.loading ? (
+    <div>loading</div>
+  ) : (
+    <div>
```

Comment:
Consider a scenario when the user has no products in their cart

Reply:
done